Portfolio/index.css

```diff
@@ -1 +1,64 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;
+          margin: 0%;
+    padding: 0%;
+}
+.cont1{
+    background-color: #90A4AE;

```

Comment:
'margin: 0%' is defined in line 4 with '*', so redefining it in '.cont1' may be unnecessary.